ChromePhp.php

```diff
@@ -244,33 +244,31 @@ public static function table()
      * internal logging call
      *
      * @param string $type
-     * @return void
+     * @return ChromePhp
      */
     protected static function _log($type, array $args)
     {
+        $logger = self::getInstance();
+
         // nothing passed in, don't do anything
-        if (count($args) == 0 && $type != self::GROUP_END) {
-            return;
+        if (empty($args) && $type != self::GROUP_END) {
+            return $logger;
         }
 
-        $logger = self::getInstance();
-
         $logger->_processed = array();
-
-        $logs = array();
-        foreach ($args as $arg) {
-            $logs[] = $logger->_convert($arg);
-        }
+        $logs = array_map(array($logger, '_convert'), $args);
```

Comment:
I think the array syntax should be short syntax because of consistency. Using the `[....]` instead.

And this short syntax is supported sine the `php-5.4+`.

Reply:
Done in PR #6.